osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -319,11 +329,29 @@ public string Text
                     }
 
                     this.FadeIn(transition_duration);
-                    textFlow.Clear();
-                    textFlow.AddText(value, s => s.TextSize = 14);
+                    addTextAsync(value);
                 }
             }
 
+            private void addTextAsync(string text)
+            {
+                var newTextFlow = new TextFlowContainer
+                {
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Colour = textFlow.Colour,
+                };
+
+                newTextFlow.AddText(text, s => s.TextSize = 14);
+
+                LoadComponentAsync(newTextFlow, d =>
+                {
+                    var textContainer = (InternalChild as FillFlowContainer);

```

Comment:
store a reference rather than casting (also never use `as` casts unless you're doing a null check on the result).